app/views/issues/_issues_table.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="table-responsive">
+  <table class="table">
+    <thead class="thead-light">
+      <tr>
+        <th><%= t('shared.sq_num_field', model: Issue.model_name.human) %></th>
+        <th><%= Issue.human_attribute_name('title') %></th>
+        <th><%= Project.model_name.human %></th>
+        <th><%= Issue.human_attribute_name('description') %></th>
+        <th><%= Issue.human_attribute_name('status') %></th>
+        <th><%= Issue.human_attribute_name('priority') %></th>
+        <th><%= Issue.human_attribute_name('issue_type') %></th>
+        <th><%= t('shared.index_views.actions') %></th>
+      </tr>
+    </thead>
+    <tbody>
+      <% issues.each do |issue| %>
+        <tr>
+          <th><%= issue.id %></th>
+          <th>
+            <%= "#{truncate(issue.title)} (" %>
+            <%= link_to "##{issue.id}", project_issue_path(issue.project.sequence_num, issue.id) %>
+            <%= ')' %>
```

Comment:
make it one line, do this for the rest as well

Reply:
done